server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ReactorNettyJaxrsServer.java

```diff
@@ -232,19 +221,14 @@ public int read() {
                    if (isTimeoutSet.get() && Exceptions.unwrap(t) instanceof TimeoutException) {
                       sendMono = resp.status(503).send();
                    } else {
+                      log.error("Unhandled server error", t);
                       sendMono = resp.status(500).send();
                    }
                    SinkSubscriber.subscribe(completionSink, sendMono);
-
-                } else {
-                   log.debug("Omitting sending back error response. Response is already committed.");
```

Comment:
I paused for a minute wondering why you removed this. Part of me feels like we should keep it, but I can't come up with a solid argument why it's more important than others that are being removed.

Reply:
Good catch.  I'm tweaking this.